test/gp/lti_sde.jl

```diff
@@ -53,23 +53,26 @@ println("lti_sde:")
     @testset "lgssm_components" begin
         rng = MersenneTwister(123456)
         N = 13
-
         kernels = vcat(
-
             # Base kernels.
-            (name="base-Matern12Kernel", val=Matern12Kernel()),
+            (name="base-Matern12Kernel", val=Matern12Kernel(), m=ZeroMean()),
```

Comment:
Rather than including the mean function inside the kernel specification in these tests, could we add an additional loop to the `@testset` below, in which a couple of means are tested?

Reply:
Done (and additionally threw a round of JuliaFormatter on the file)